examtex

```diff
@@ -24,7 +24,7 @@ fi
 # save current directory
 cwd=$(pwd)
 # change directory to source
-cd "$(dirname "$(readlink -f "$0")")"
+cd "$(dirname "$(greadlink -f "$0")")"
```

Comment:
OSX `readlink` doesn't have the `-f` flag :( maybe you can do some detection or something but also not everyone has `greadlink`